cluster-sync/external/README.md

```diff
@@ -34,3 +34,16 @@ export KUBEVIRT_PROVIDER=external
 The external provider isn't quite appropriate for use with the local-up-cluster.sh script used 
 in the Kubernetes source repo.  We'll need to add an additional `local` provider for this to 
 handle some of the specifics.
+
+# External Openshift provider
+
+In addition to setting the KUBEVIRT_PROVIDER=external, also set the EXTERNAL_PROVDER=openshift. This will call additional paths to push properly to openshift registry, and expose the correct ports to do so. The MachineConfigOperator is used to patch the insecure registries to include the registry we added. If the MachineConfigOperator is not available (CodeReadyContainers for instance) you have to configure the insecure registries ahead of time or the cluster-sync will fail.
```

Comment:
EXTERNAL_PROVDER -> EXTERNAL_PROVIDER

Reply:
Removed the whole text as its no longer needed.